score/scripts/download_cbg.py

```diff
@@ -3,11 +3,15 @@
 import zipfile
 import os
 import json
+from pathlib import Path
+
+data_path = Path.cwd().joinpath("data")
```

Comment:
Even cooler pathlib does operator overloading with "/" so you can do 
```python
Path.cwd() / 'data'
```
optional though

Reply:
Oh, that would be much cleaner, let me do it quick.